sonar-check-api/src/main/java/org/sonar/check/Priority.java

```diff
@@ -19,6 +19,10 @@
  */
 package org.sonar.check;
 
+/**
+ * @deprecated since 5.6 sonar-check-api is deprecated 
+ */
+@Deprecated
 public enum Priority {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)